omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -44,7 +44,11 @@
       default_type text/plain;
       alias /opt/opscode/version-manifest.txt;
     }
-
+    # location for Let's Encrypt challenges
+    location /.well-known/acme-challenge {
```

Comment:
@srenatus that is a good call, using the the lets_encrypt_enable flag will make it cleaner. In the future, you might consider a deeper integration (managing the validation and certs all together) and this flag will be handy. I will add it in.